Documentation/Home/ConfiguringTYPO3.rst

```diff
@@ -28,9 +28,7 @@ Configuring TYPO3
 
             .. rst-class:: horizbuttons-striking-m
 
-            -  `12-dev <https://docs.typo3.org/m/typo3/reference-coreapi/main/en-us/ApiOverview/Configuration/Index.html>`__
             -  `11.5 <https://docs.typo3.org/m/typo3/reference-coreapi/11.5/en-us/ApiOverview/Configuration/Index.html>`__
```

Comment:
12-dev is removed, but 12.4 not added. Is there a reason? Also for the other cards below.

Reply:
Good point Chris